.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Daffodil Schema Template CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Java ${{ matrix.java_version }}, Scala ${{ matrix.scala_version }}, ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        java_version: [ 8, 11, 15 ]
+        scala_version: [ 2.12.11 ]
+        os: [ 'ubuntu-latest' ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+      ############################################################
+      # Setup
+      ############################################################
+
+      - name: Checkout Repository
+        uses: actions/checkout@v2.0.0
+
+      - name: Install Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java_version }}
+
+      ############################################################
+      # Test
+      ############################################################
+
+      - name: Run Rat Check
+        run: sbt ++${{ matrix.scala_version }} ratCheck
+
+      - name: Test
+        run: sbt ++${{ matrix.scala_version }} test
```

Comment:
Is this how the `src/test/g8/test` file gets used?  Might want to change name to `name: Run Tests`.

Are you enabling GitHub Actions?  This pull request didn't run any CI job after you created it.

Reply:
Yeah, this "sbt test" will run the scripted g8 test mentioned before. Ill rename the action to make it more clear.

I think maybe the GitHub action won't be enabled until it's in the master branch, maybe? You can look see what it will look like on my fork: https://github.com/stevedlawrence/daffodil-schema.g8/runs/1982074448